tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php

```diff
@@ -104,6 +140,17 @@ public function testNodeFromRegistry()
         $this->subscriber->handleNodeFromRegistry($this->persistEvent->reveal());
     }
 
+    public function testReorderNodeFomRegistry()
```

Comment:
In anycase, this is how the current code has been written. I did not
write it like this for fun. This approach will ensure that people can
understand the tests and the purpose of each test. Please add the
comment.

On Tue, Jul 21, 2015 at 01:13:11AM -0700, Daniel Rotter wrote:
>    In [1]tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php:
> 
>  > @@ -104,6 +140,17 @@ public function testNodeFromRegistry()
>  >          $this->subscriber->handleNodeFromRegistry($this->persistEvent->reveal());
>  >      }
>  >
>  > +    public function testReorderNodeFomRegistry()
> 
>    [2]@chirimoya What's your opinion on that?
> 
>    —
>    Reply to this email directly or [3]view it on GitHub.
> 
>    Reverse link: [4]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu-document-manager/pull/28#discussion_r35078475
>    2. https://github.com/chirimoya
>    3. https://github.com/sulu-io/sulu-document-manager/pull/28/files#r35078475
>    4. https://github.com/sulu-io/sulu-document-manager/pull/28/files#r35078475